src/modules/buildingHover/main.ts

```diff
@@ -67,7 +67,11 @@ export default (async (LSSM, MODULE_ID) => {
                 }&nbsp;<i class="fa fa-users"></i>&nbsp;${
                     building.personal_count
                 }`;
-                if (building.building_type === 6)
+                if (
+                    Object.values(LSSM.$t('cellBuildings')).indexOf(
+                        building.building_type
+                    ) >= 0
+                )
```

Comment:
This does work but I would prefer using newer Code with .includes function:
```suggestion
if(Object.values(LSSM.$t('cellBuildings')).includes(building.building_type))
```